### PR TITLE
NDS-412 design system update action banner component

### PR DESCRIPTION
**Pull request by johndavey72**
https://nicedigital.atlassian.net/browse/NDS-412

### CODE REVIEW

**Review by barkertron — DISMISSED** [review]

**Review by barkertron — APPROVED** [review]

File: components/nds-action-banner/scss/action-banner.scss [hunk 1]
```diff
@@ -54,57 +54,144 @@
...
 
-  &--subtle {
+  &--full-width {
+    @include layout.fullWidth();
     @include typography.links-default;
-    background: colours.$action-banner-subtle-background;
-    border: 1px solid colours.$action-banner-subtle-border;
-    color: colours.$text;
+    background: colours.$action-banner-full-width-background;
+    border: 1px solid colours.$action-banner-full-width-border;
```
[the remark this answers is not in the document]
Comment (johndavey72) [reply components/nds-action-banner/scss/action-banner.scss:76]: Me neither, good spot. I've removed the border. These styles were lifted and shifted from the hero component so it's likely there is some cruft in there.

File: components/nds-action-banner/scss/action-banner.scss [hunk 2]
```diff
@@ -54,57 +54,144 @@
...
+  &--subtle {
+    @include typography.links-default;
+    background: colours.$action-banner-subtle-background;
+    border: 1px solid colours.$action-banner-subtle-border;
+    color: colours.$text;
 
-    &:hover {
-      background: rgb(0 0 0 / 20%);
+    .action-banner__title {
+      color: colours.$text;
```
[the remark this answers is not in the document]
Comment (johndavey72) [reply components/nds-action-banner/scss/action-banner.scss:64]: Good point

File: components/nds-action-banner/scss/action-banner.scss [hunk 3]
```diff
@@ -54,57 +54,144 @@
...
+    color: colours.$text-inverse;
+    padding: 0;
+
+    .action-banner__intro * {
+      @include typography.lead();
+      margin-top: utils.rem(spacing.$medium);
+      margin-bottom: utils.rem(spacing.$large);
+    }
+
+    .action-banner__container {
```
[the remark this answers is not in the document]
Comment (johndavey72) [reply components/nds-action-banner/scss/action-banner.scss:86]: Breaks the styling if I apply them there unfortunately, assuming I've understood correctly.

File: components/nds-action-banner/scss/action-banner.scss [hunk 4]
```diff
@@ -54,57 +54,144 @@
...
+        outline-color: colours.$focus-inverse;
+      }
+    }
+  }
+
+  &--full-width-subtle {
+    @extend .action-banner--full-width;
+
+    background: colours.$action-banner-full-width-subtle-background;
+    border: 1px solid colours.$action-banner-full-width-subtle-border;
```
[the remark this answers is not in the document]
Comment (johndavey72) [reply components/nds-action-banner/scss/action-banner.scss:134]: Removed the alias token also.

File: components/nds-action-banner/src/ActionBanner.tsx [hunk 5]
```diff
@@ -1,86 +1,53 @@
-import React, { ReactNode } from "react";
+import React, { ReactNode, useState } from "react";
```
[the remark this answers is not in the document]
Comment (johndavey72) [reply components/nds-action-banner/src/ActionBanner.tsx:1]: Yep you're right :-)

File: docs/pages/testpage.tsx
[the remark this answers is not in the document]
Comment (johndavey72) [reply docs/pages/testpage.tsx:1]: Wish I'd spotted that before getting the red build on TC. I've wrapped the children with the component now, just my lazy shorthand :-)

**Pull request merged**